### PR TITLE
Fix missing include to unbreak build.

### PR DESCRIPTION
**Pull request by hzeller**
Fix broken build not finding fprintf() due to missing stdio.h include.

Signed-off-by: Henner Zeller <h.zeller@acm.org>

### CODE REVIEW

File: libs/libvtrutil/src/vtr_geometry.h [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #include "vtr_range.h"
 #include "vtr_assert.h"
 
+#include <stdio.h> // vtr_geometry.tpp uses printf()
```
Comment (litghost) [inline libs/libvtrutil/src/vtr_geometry.h:6]: Should probably be `cstdio`?

**Review by litghost — APPROVED** [review]
LGTM

File: libs/libvtrutil/src/vtr_geometry.h (hunk 1 above)
Comment (hzeller) [reply libs/libvtrutil/src/vtr_geometry.h]: Looks like it is common in this directory, so let's use `<cstdio>`. Updated.
(usually, I recommend `stdio.h` for cleanly stating the origins, but seem that this part of the project is not doing that)

Comment (litghost) [reply libs/libvtrutil/src/vtr_geometry.h]: In C++, `cstdio` should always be preferred, at least by the standard: https://stackoverflow.com/questions/7596406/stdio-h-not-standard-in-c

Comment (hzeller) [reply libs/libvtrutil/src/vtr_geometry.h]: Yes, but it is important to know the rules to know when it is ok and better to break them :D

**Pull request merged**